applications/app/controllers/ShortUrlsCampaignController.scala

```diff
@@ -0,0 +1,51 @@
+package controllers
+
+import common.Logging
+import common.`package`._
+import conf.LiveContentApi
+import model.Cached
+import play.api.mvc.{RequestHeader, Action, Controller}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object ShortUrlsCampaignController extends Controller with Logging {
+
+  val campaigns = Map (
+    "iw" -> "twt_ipd",
+    "wc" -> "twt_wc",
+    "tf" -> "twt_fd",
+    "fb" -> "fb_gu",
+    "fo" -> "fb_ot",
+    "us" -> "fb_us",
+    "au" -> "soc_567",
+    "tw" -> "twt_gu",
+    "at" -> "twt_atn",
+    "sfb"-> "share_btn_fb",
+    "ip" -> "twt_iph",
+    "stw" -> "share_btn_tw",
+    "swa" -> "share_btn_wa",
+    "em" -> "email",
+    "sgp" -> "share_btn_gp",
+    "sbl" -> "share_btn_link"
+  )
+
```

Comment:
In the R2 Code db I couldn't see any set to internal campaigns. I will double check on Prod db though I think Code is fairly up to date.